omod/src/main/compass/sass/login.scss

```diff
@@ -8,4 +8,9 @@
       width: 405px;
     }
   }
-}
\ No newline at end of file
+}
+
+.location-error {
+	color: red;
```

Comment:
Thanks. Am building and launching the ref app now to test. will push changes in a few minutes. :)